tests/phpunit/Mutator/Number/DecrementIntegerTest.php

```diff
@@ -476,6 +476,20 @@ public static function mutationsProvider(): iterable
                 PHP,
         ];
 
+        // see https://ieeexplore.ieee.org/stamp/stamp.jsp?tp=&arnumber=9524503 "Suppressing Unproductive Mutants"
+        yield 'It does not decrement when it is used in sleep functions' => [
+            <<<'PHP'
+                <?php
+                usleep(1_000);
+                sleep(1_000);
+                time_nanosleep(1_000, 0);
+                set_time_limit(1_000);
+
+                ini_set('memory_limit', 500); // in bytes
+                ini_set('memory_limit', -1);
+                PHP,
+        ];
+
         yield 'It does not decrement limit argument of preg_split function when it equals to 0' => [
```

Comment:
I think we should also add a test for a method called `sleep` and takes an `int` but will not be mutated because its not a `FuncCall`